src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditCopyUiTest.java

```diff
@@ -98,7 +98,7 @@ public void allTests() throws Exception{
         feedbackEditPage.fsCopyToModal.clickSubmitButton();
         feedbackEditPage.waitForPageToLoad();
```

Comment:
We might be able to remove waitForPageToLoad, if using verifyStatusWithRetry is the way to solve this issue. I'm guessing the page load flag in selenium doesn't reset when the page's location is modified using javascript